src/SFML/Graphics/Shader.cpp

```diff
@@ -228,6 +229,14 @@ m_uniforms      ()
 }
 
 
+////////////////////////////////////////////////////////////
+Shader::Shader(Shader&& other) :
+Shader()
+{
+    *this = std::move(other);
```

Comment:
You're using operator assignment inside constructor logic which loses the benefits of in-place construction.

Solution: Definitions are not needed when the declaration states `= default`